### PR TITLE
Create item moving helper class

### PR DESCRIPTION
**Pull request by ReinforceZwei**
From #1208

Also found a bug (i guess) while testing new class. Player inventory isn't available after the `OnAfterGameJoined` event.

### CODE REVIEW

File: MinecraftClient/Inventory/ItemMovingHelper.cs [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+namespace MinecraftClient.Inventory
+{
+    /// <summary>
+    /// Class that contains useful methods to move item around in a container
+    /// </summary>
+    public class ItemMovingHelper
+    {
+        private Container c;
+        private Func<int, int, WindowActionType, bool> windowAction;
```
Comment (ORelio) [inline MinecraftClient/Inventory/ItemMovingHelper.cs:15]: Any reason for storing a callback to windowAction here and not just a reference to McClient itself? 🤔

Comment (ReinforceZwei) [reply MinecraftClient/Inventory/ItemMovingHelper.cs:15]: Yes. Since ChatBot cannot access to McClient

Comment (ORelio) [reply MinecraftClient/Inventory/ItemMovingHelper.cs:15]: Oh, OK. In that case, how adding this to the ChatBot API just after the `WindowAction` API method?
```C#
        /// <summary>
        /// Get inventory action helper
        /// </summary>
        /// <param name="container">Inventory Container</param>
        /// <returns>ItemMovingHelper instance</returns>
        protected ItemMovingHelper GetItemMovingHelper(Container container)
        {
            return new ItemMovingHelper(Handler, container);
        }
```
Would make the ItemMovingHelper more visible and easy to use for people using the ChatBot API.

Comment (ReinforceZwei) [reply MinecraftClient/Inventory/ItemMovingHelper.cs:15]: Okay.

**Review by ORelio — APPROVED** [review]

**Pull request merged**